test/edu/ucsb/nceas/metacat/doi/osti/OstiDOIServiceIT.java

```diff
@@ -55,25 +63,47 @@ public class OstiDOIServiceIT {
      */
     @Before
     public void setUp() throws Exception {
-        LeanTestUtils.initializePropertyService(LeanTestUtils.PropertiesMode.LIVE_TEST);
         d1NodeTest = new D1NodeServiceTest("initialize");
+        final String passwdMsg =
+                """
+                \n* * * * * * * * * * * * * * * * * * *
+                DOI CREDENTIALS NOT SET!
+                Test requires OSTI-specific values for
+                'guid.doi.username' & 'guid.doi.password'
+                in your test/test.properties file!
+                * * * * * * * * * * * * * * * * * * *
+                """;
+        Properties testProperties = LeanTestUtils.getExpectedProperties();
+        String ostiName = testProperties.getProperty("guid.doi.username");
```

Comment:
Better to catch the PropertyNotFound exception as shown in my suggested snippet, otherwise the setUp method will throw an exception without any explanation, if the property is not in the test props file

Reply:
This statement is calling a java object `Properties`. So it will not throw a PropertyNotFoundException :( . According to the java doc, it will return null if the property doesn't exist. So I check if `ostiName` is null with the error message.